cli/api.js

```diff
@@ -5,7 +5,12 @@ const registryUrl = require('registry-url')();
 const pify = require('pify');
 const recast = require('recast');
 
-const fileName = `${os.homedir()}/.hyper.js`;
+const devConfigFileName = '../.hyper.js';
```

Comment:
You need to resolve this path with something like [__dirname](https://nodejs.org/docs/latest/api/modules.html#modules_dirname). It should not be impacted by the current working directory.